### ISSUE TITLE
Loading GPU Trained Model on CPU

### ISSUE THREAD

**Issue by ilkarman** [dmlc/mxnet#5046]
I load the trained resnet model obtained [here](http://data.mxnet.io/models/imagenet-11k/resnet-152/) on a pre-built MXNet 0.7.0 obtained from [here](https://github.com/dmlc/mxnet/releases/tag/20160531) - 20160531_win2012_x64_gpu.7z.

I am using Azure Web App which runs Windows and Python 2.7 (x64). Initially I can't load the model until I remove all the cudnn lines from the json:

```
      "op": "Convolution",
      "param": {
        "cudnn_tune": "limited_workspace",
```

However, after this the model (loads but) takes a long time to run a prediction and always returns the same result regardless of input image:

`probability=0.007025, class=n04351699 suiting`

I then take the model and symbol file; along with the same code and run it on a linux-machine; in the same way (CPU):

`model = mx.model.FeedForward.load("resnet-152", 0, ctx=mx.cpu())`

And it seems to run fine.

**Comment by piiswrong** [comment]
Windows binary release has windows/cuda version restrictions

**Comment by ilkarman** [comment]
Fixed this by upgrading to latest version (0.9) - didn't need to remove the cudnn_tune also
